include/rocksdb/db.h

```diff
@@ -818,10 +818,10 @@ class DB {
   //        even if the file compression dont match the level compression
```

Comment:
Let's update the comments of the function to include the new option, and the implications of using it